cmd/gpud/command/status.go

```diff
@@ -56,7 +51,9 @@ func cmdStatus(cliContext *cli.Context) error {
 	fmt.Printf("%s successfully checked gpud health\n", checkMark)
 
 	for {
-		packageStatus, err := getStatus()
+		cctx, ccancel := context.WithTimeout(rootCtx, 15*time.Second)
+		packageStatus, err := client.GetPackageStatus(cctx, fmt.Sprintf("https://localhost:%d/admin/packages", config.DefaultGPUdPort))
```

Comment:
A URI like `/admin/packages` should be defined in the apis package.

Reply:
Kept it in `/internal/server` for now, we can refactor API paths in a separate PR /cc @cardyok

Reply:
sgtm, we do not have remote dependencies for this api now.